jiant/scripts/preproc/superglue_leaderboard_submission_formatter.py

```diff
@@ -0,0 +1,52 @@
+"""Translate raw prediction files for SuperGLUE tasks into format expected by SuperGLUE leaderboard.
+
+This script helps translate the raw prediction files for SuperGLUE tasks into the jsonl files
```

Comment:
Can _helps_ be removed here? Unless there is something else required, in which case it should be specified.

Reply:
Thanks, done w/ 12e65e4.